lib/tests/2wp.js

```diff
@@ -139,7 +139,7 @@ const execute = (description, getRskHost) => {
         const senderRecipientInfo = await createSenderRecipientInfo(rskTxHelper, btcTxHelper);
         const initialSenderAddressBalanceInSatoshis = await getBtcAddressBalanceInSatoshis(btcTxHelper, senderRecipientInfo.btcSenderAddressInfo.address);
         const peginValueInSatoshis = minimumPeginValueInSatoshis;
-        const peginV1RskRecipientAddress = await rskTxHelper.newAccountWithSeed('successfulPeginV1');
+        const peginV1RskRecipientAddress = await rskTxHelper.newAccountWithSeed(rskTxHelper.getClient().utils.randomHex(16));
```

Comment:
Why this change is needed? Didn't we agree on making tests as more deterministic as possible?